### ISSUE TITLE
how's your model converge ?

### ISSUE THREAD

**Issue by jinfagang** [FateScript/CenterNet-better#6]
From my experiments on single GPU the model impossible to converge.

I have changed learing from 0.02 to 0.01, also 0.0025, if I keep 0.02 the loss will became Nan quickly, I assume you using 0.02 for 8 GPUs defaultly.

My question is that, in whatever minimal learing rate the model impossible to cnverge on single GPU. After 9999 iterations the model loss still very high and the loss like this:

```
 iter: 9999  total_loss: 6.486  loss_cls: 4.432  loss_box_wh: 1.896  loss_center_reg: 0.260  tim: 0.7158  data_tim: 0.2920  lr: 0.007393  mem: 5030M
```
How's your learning curve on 8 GPUs? What's the loss going to be?

**Comment by FateScript** [comment]
> question is that, in whatever minimal learing rate t

On 8-GPU，its loss will converge and the final loss value will be about 1 or less. I nerver tried 1-GPU version, but the loss is somehow hard to converge with small learning rate. 

Maybe the optimization curve is strange and when it achieve about 6, you should try a big lr to avoid a local minimum.

**Comment by jinfagang** [comment]
I managed using resnet50 and resnet101 backbone train.

loss seems normal now, while dcn based head hard to converge.
